src/alt/index.js

```diff
@@ -46,6 +46,15 @@ class Alt {
   }
 
   createStore(StoreModel, iden, ...args) {
+    // allow hot loading for development usage
+    if (process.env.NODE_ENV !== 'production') {
+      if (module.hot) {
```

Comment:
Yeah, that's possible. I just ported the code from the current hot loading utility and didn't really think about it in detail.

Feel free to do whatever changes you feel necessary. One problem with the current PR is that it introduces a code path that's not tested (-> lower coverage).